docs/sources/api/_index.md

```diff
@@ -1044,7 +1048,13 @@ curl -g -X POST \
   -H 'X-Scope-OrgID: 1'
 ```
 
-The query parameter can also include filter operations. For example `query={foo="bar"} |= "other"` will filter out lines that contain the string "other" for the streams matching the stream selector `{foo="bar"}`.
+The same example deletion request for Grafana Enterprise Logs uses Basic Authentication and specifies the tenant name as a user; `Tenant1` is the tenant name in this example. The password in this example is an access policy token that has been defined in the API_TOKEN environment variable. The token must be for an access policy with `logs:delete` scope for the tenant specified in the user field:
+
+```bash
+curl -u "Tenant1:$API_TOKEN" \
+  -g -X POST \
+  'http://127.0.0.1:3100/loki/api/v1/delete?query={foo="bar"}&start=1591616227&end=1591619692'
```

Comment:
We've found out that running these kinds of post requests in bash causes problems with escaping the `"`, '|` and so on characters. The `--data-urlencode` parameter for curl is preferred. https://github.com/grafana/deployment_tools/blob/master/docs/loki/support-escalation.md#manual-operational-tasks has an example. Perhaps we should use that format.